Fixed an `AttributeError` raised by running a single testcase on interactive UI with an active filter.